languages.ts

```diff
@@ -552,4 +552,38 @@ export const languageSpecs: LanguageSpec[] = [
         },
         stylized: 'R',
     },
+    {
+        handlerArgs: {
+            languageID: 'pascal',
```

Comment:
When running locally, Pascal code intel doesn't work. I suspect it's because there's no Pascal entry in https://github.com/sourcegraph/sourcegraph/blob/3a1dd8ccd49175ef2ed47dabe85d2e0543ce6c47/shared/src/languages.ts#L40

Reply:
Yes, in my local testing I also have https://github.com/sourcegraph/sourcegraph/pull/8079 patched in. That _should_ give you working results, but please let me know if it does not.